examples/complete/main.tf

```diff
@@ -2,14 +2,21 @@ provider "aws" {
   region = var.region
 }
 
+provider "github" {
+  token = var.github_token
+  owner = var.owner
+}
+
 module "webhooks" {
   source              = "../../"
   active              = false
-  github_token        = var.github_token
-  github_organization = var.github_organization
   github_repositories = var.github_repositories
   webhook_url         = var.webhook_url
   events              = var.events
 
   context = module.this.context
+
+  providers = {
+    github = github
+  }
```

Comment:
This is not needed since there isn't an alias, the provider should be implicitly provided